configure/executors/dind/executor.Deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     spec:
       containers:
         - name: executor
-          image: index.docker.io/sourcegraph/executor:insiders@sha256:da6653e0cd6f95d60a0db2f0342c4d77d652d786112dbb8d66752e68f1e0609d
+          image: index.docker.io/sourcegraph/executor:insiders@sha256:973d40de6e9a5ac27341d9a1eafd8383705eb09d2647de543775abe91da69779
```

Comment:
@BolajiOlajide @eseliger intentional that we run insiders here? Same for the k8s/executor.

Reply:
Naaah, I'm not sure. 

@jdpleiness @DaedalusG Y'all know about this?

Reply:
Nah, I don't know of any reason this should be insiders.